packages/mobile/src/components/premium-track-purchase-drawer/StripePurchaseConfirmationButton.tsx

```diff
@@ -0,0 +1,48 @@
+import { useCallback } from 'react'
+
+import { Button } from 'app/components/core'
+import { useNavigation } from 'app/hooks/useNavigation'
+import { createStripeSession } from 'app/services/buyCrypto'
+import { useThemeColors } from 'app/utils/theme'
+
+const messages = {
+  buy: (price: string) => `Buy $${price}`
+}
+
+type StripePurchaseConfirmationButtonProps = {
+  price: string
+}
+
+export const StripePurchaseConfirmationButton = ({
+  price
+}: StripePurchaseConfirmationButtonProps) => {
+  const navigation = useNavigation()
+  const { specialLightGreen1 } = useThemeColors()
+
+  const handleBuyPress = useCallback(async () => {
+    try {
+      const res = await createStripeSession({
```

Comment:
So, isn't there already a function in web that does this? I was hoping we could move things like that to common.

Reply:
Agree. There is an equivalent fn in web, but libs there is attached to window whereas in mobile it's not. We _could_ pass in libs I guess but that feels a bit weird/not sure if that's a pattern we want to follow. Any thoughts @rickyrombo ?

Reply:
need to think on this one more. I think what we have here is good for now, but we definitely need to establish a "meta" for libs/sdk